File: generated-sources/csharp-dotnet2/mojang-api/src/main/CsharpDotNet2/Org/OpenAPITools/Model/OrderStatisticsRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// The body of the request to get the order statistics
  /// </summary>
  [DataContract]
  public class OrderStatisticsRequest {
    /// <summary>
    /// Gets or Sets MetricKeys
    /// </summary>
    [DataMember(Name="metricKeys", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "metricKeys")]
    public List<OrderStatistic> MetricKeys { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class OrderStatisticsRequest {\n");
      sb.Append("  MetricKeys: ").Append(MetricKeys).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: generated-sources/dart/mojang-api/lib/model/order_statistics_request.dart
part of openapi.api;

class OrderStatisticsRequest {
  
  List<OrderStatistic> metricKeys = [];
  OrderStatisticsRequest();

  @override
  String toString() {
    return 'OrderStatisticsRequest[metricKeys=$metricKeys, ]';
  }

  OrderStatisticsRequest.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    metricKeys = OrderStatistic.listFromJson(json['metricKeys']);
  }

  Map<String, dynamic> toJson() {
    return {
      'metricKeys': metricKeys
    };
  }

  static List<OrderStatisticsRequest> listFromJson(List<dynamic> json) {
    return json == null ? new List<OrderStatisticsRequest>() : json.map((value) => new OrderStatisticsRequest.fromJson(value)).toList();
  }

  static Map<String, OrderStatisticsRequest> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, OrderStatisticsRequest>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new OrderStatisticsRequest.fromJson(value));
    }
    return map;
  }
}

